docs/_articles/ja/apex/apex-get-started.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Quick Start Apex Development with Salesforce Extensions
+lang: en
+---
+  
+## Quick Start Apex Development
+Complete this Quick Start to create, test and debug a workflow that is created using an Apex class, and sets off a Apex trigger.
+
+We’ll create a custom object called `Book`, and add a custom field called `Price` to that custom object. We’ll update and test the updated Price field using Apex and learn how to work with Apex Code using the Salesforce Extensions.
+
+
+Before you start, install [Salesforce Extensions for Desktop](./en/../../vscode-desktop/install.md) or [Set Up Code Builder](./en/../../codebuilder/cb-setup.md)
+
+
+### Create a Custom Object
+1. Log in to your sandbox or Developer org.
```

Comment:
```suggestion
1. Log in to your sandbox or developer org.
```

Reply:
If you're using developer org generically, this I would l/c it. If you're specifically referring to a Developer Edition org, then I'd cap it.